docs/decisions/2023-06-05_github_merge_queue.md

```diff
@@ -0,0 +1,49 @@
+# 2023-06-05 GitHub Merge Queue
```

Comment:
Will this be removed before merging and merged in [6581](https://github.com/evergreen-ci/evergreen/pull/6581/files) instead?

Reply:
thank you!